numpy/add_newdocs.py

```diff
@@ -463,6 +463,43 @@ def luf(lamdaexpr, *args, **kwargs):
 
     """))
 
+add_newdoc('numpy.core', 'nested_iters',
+    """
+    Convenience function to create multiple nditers from a single ndarray.
+
+    Parameters (see nditer)
+    ---------
+    op : ndarray
+        The array to iterate over.
+
+    axes :  a sequence of at least 2 tuples of itegers, as nditer "op_axes"
```

Comment:
Couple remarks:

* Typo
* Claims to need tuples, but the example below gives `list`s
* This should be divided into something type-like, and a description. So maybe `list of tuple` after the colon. Same below.

Reply:
part 1,2 fixed in ffedc25, by "same below" do you mean I should copy the parameter definition for each of the optional parameters from nditer?

Reply:
Actually, the error messages given by `nested_iters` want a tuple of tuples

Reply:
By same below, I was just referring to the type / description separation. Leaving the type below blank seems fine.